src/main/java/com/theateam/waaprojectspringbackend/controller/CustomersController.java

```diff
@@ -53,15 +60,45 @@ public void updateOffer(@PathVariable Long offerId, @RequestBody UpdateOfferDto
     }
 
     @GetMapping("/saved-properties")
-    public String getSavedProperties() {
-        return "Greetings from Customers GET saved-properties!";
+    public ResponseEntity<List<String>> getSavedProperties() {
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        String userEmail = authentication.getName();
+        Optional<User> user = userService.getUserByEmail(userEmail);
+
+        if(user!=null){
+            List<String> savedPropertyIds = user.get().getSavedPropertyIds();
+            return ResponseEntity.ok(savedPropertyIds);
+        }else {
+            return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(null);
+        }
     }
 
-    @PostMapping("/saved-properties")
-    public String savedProperties() {
-        return "Greetings from Customers POST saved-properties!";
+    @PostMapping("/saved-properties/{propertyId}")
```

Comment:
route should be /saved-properties and propertyId should be sent in body. Here is accessed saved properties list, and a property added to it, not to the property. so POST /saved-properties @RequestBody propertyId